source/nodejs/adaptivecards/src/__tests__/components/image.spec.ts

```diff
@@ -5,4 +5,5 @@ import {Image} from "../../card-elements";
 test('Image should be instantiated', ()=>{
     const image = new Image();
     expect(image).toEqual(expect.anything());
+    expect(image.altText).toBeUndefined();
```

Comment:
I know you were working on more comprehensive UTs for the platform.. so potentially some coming up which actually test the altText flowing through if specified/not undefined?